data/datafile.go

```diff
@@ -450,7 +450,7 @@ func appAssetsJsAppJs() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "app/assets/js/app.js", size: 43468, mode: os.FileMode(420), modTime: time.Unix(1450202755, 0)}
+	info := bindataFileInfo{name: "app/assets/js/app.js", size: 43468, mode: os.FileMode(416), modTime: time.Unix(1454014512, 0)}
```

Comment:
any reason that these file perms are expressed as decimal instead of something more easily understood (e.g. octal `0640`)?